src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/Application.js

```diff
@@ -14,6 +14,8 @@ import ViewRenderer from '../ViewRenderer';
 import userStore from '../../stores/UserStore';
 import {Backdrop} from '../../components';
 import Login from '../Login';
+import ProfileFormOverlay from '../ProfileOverlay/ProfileFormOverlay';
+import Loader from '../../components/Loader/Loader';
```

Comment:
And in general the ones with a higher count of `..` in the beginning should go first. Sadly the eslint rule is not so strict about it as I wish it would be...